app/assets/stylesheets/target.scss

```diff
@@ -0,0 +1,293 @@
+// Place all the styles related to the Target controller here.
```

Comment:
Look into more semantic names for your ids. Also it seems you are using lots if IDs vs css classes to reuse common styles.

Reply:
Following internal css guidelines please avoid using ids (https://cssguidelin.es/)